apps/i18n/tutorialExplorer/en_us.json

```diff
@@ -44,11 +44,13 @@
   "filterHeaderDefault": "Sort",
   "filterHeaderDisplayWeight": "Top rated",
   "filterHeaderPopularityRank": "Most popular",
-  "roboticsButtonText": "Got robots? <a href='{robotics_url}'>Use these activities</a> and make a tangible Hour of Code for students of any age!",
+  "roboticsButtonText": "View robotics activities",
+  "roboticsText": "Got robots? <a href='{robotics_url}'>Use these activities</a> and make a tangible Hour of Code for students of any age!",
   "tutorialDetailsTeacherNotes": "Teacher notes",
   "tutorialDetailsShortLink": "Short link",
   "tutorialDetailInternationalLanguages": "Languages",
+  "tutorialDetailStandards": "Standards",
   "headingTutorialsYourLanguage": "Tutorials in your language",
   "headingTutorialsManyLanguages": "Tutorials in many languages",
-  "tutorialDetailStandards": "Standards"
+  "bottomGuidelinesLink": "To see our criteria and submission guidelines for Hour of Code tutorials, <a href='{guidelines_url}'>click here</a>."
```

Comment:
My understand of loc is that we always need to keep the entire sentence in a single string, because we don't know how other languages might order things grammatically.  So we can't do things like have separately translated sentence partials, and then just append them afterwards.  The only reason we're using tags inside the string is because they have links for just a part of the sentence.  @tanyaparker can confirm.